packages/expo-cellular/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### 🛠 Breaking changes
 
+- Replace static fields with async methods([#13729](https://github.com/expo/expo/pull/13729) by [@m1st4ke](https://github.com/m1st4ke))
+
 ### 🎉 New features
 
```

Comment:
I'd add another entry listing all new methods, to emphasize that they were added 😉